splunk_add_on_ucc_framework/commands/rest_builder/endpoint/base.py

```diff
@@ -47,10 +51,15 @@ class RestEntityBuilder:
 """
 
     def __init__(
-        self, name: Optional[str], fields: List["RestFieldBuilder"], **kwargs: Any
+        self,
+        name: Optional[str],
+        fields: List["RestFieldBuilder"],
+        special_fields: List["RestFieldBuilder"],
```

Comment:
In splunktaucclib you had made this parameter of type `Optional[List[RestField]]` should we mention the same thing here.